lib/messages/blob.js

```diff
@@ -39,7 +40,7 @@ class Blob extends Message {
 
     const tripleBackTick = '```';
     return {
-      fallback: title,
+      fallback: `[${this.repository.full_name}] - ${title}`,
```

Comment:
We don't ever post an activity message for blobs, so would this ever actually show up in a notification? I'm not sure how the fallback works for unfurls, but I have a feeling the Slack notification is just the link that the user posted.